### PR TITLE
Misc refactoring

### PR DESCRIPTION
**Pull request by ghost**
Just some minor changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,10 +64,17 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
...
-Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/sm_app_config. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](contributor-covenant.org) code of conduct.
+1. Fork it (https://github.com/SouthernMade/sm_app_config/fork)
+2. Create your feature branch (`git checkout -b feature/my-new-feature`)
+3. Write tests for your feature, or regression tests highlighting a bug
+4. Write the feature itself, or fix the bug
+5. Commit your changes (`git commit -am 'Add some feature'`)
+6. Push the branch (`git push origin feature/my-new-feature`)
+7. Create a new Pull Request
 
+This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Code of Conduct](CODE_OF_CONDUCT.md).
```
Comment (mattmueller) [inline README.md:75]: Prolly need an actual code of conduct if we link one.

Comment (ghost) [inline README.md:75]: https://github.com/SouthernMade/sm_app_config/blob/master/CODE_OF_CONDUCT.md

Comment (mattmueller) [inline README.md:75]: lol im dumb kthx.

Comment (ghost) [inline README.md:75]: Noted. :smiling_imp:

**Comment by jwhitis** [comment]
@jramos All good, thanks.

**Pull request merged**